lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -55,7 +55,13 @@ def add_title_and_description_fallbacks!
           msg << "Jekyll GitHub Metadata will not set site.title to the repository's name."
           Jekyll::GitHubMetadata.log :warn, msg
         else
-          site.config["title"] ||= Value.new("title", proc { |_c, r| r.name })
+          site.config["title"] ||= Value.new("title", proc { |_context, repository|
+            if repository.project_page?
+              repository.name # Someday if GitHub supports display names for repos, prefer it here.
```

Comment:
Rather remove this TODO.

It's clear from the implementation that only `name` is being used not a display name like below.

If display name ever became a thing, there's be an issue to investigate areas to use it and this area of `repository.name` should come up easily without the comment adding clutter.